.changeset/fuzzy-papayas-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'doctocat-nextjs-site': patch
+---
+
+Removed `_meta.global.ts` and instead directly pass header and sidebar links into the doctocat `Theme` component.
```

Comment:
Looks good, but could we please add an example of what this should look like for users? 

Something like:




```diff 
- // _meta.global.ts
- export default {
-   sidebarLink: {
-     type: 'page',
-     href: 'https://github.com/primer/doctocat-nextjs',
-     title: 'Doctocat',
-   }
- }
```

followed by

```diff 
+ // app/layout.tsx
+ 
+ const sidebarLinks: ThemeProps['sidebarLinks'] = [
+   {
+     href: 'https://github.com/',
+     title: 'GitHub',
+     isExternal: true,
+   },
+ ]
+ 
+ <Theme sidebarLinks={sidebarLinks} {...rest} />
```